src/lib/gtfs2geojson.js

```diff
@@ -0,0 +1,82 @@
+// vendored gtfs2geojson because the latest published version requires `dsv` which
```

Comment:
Should the license be inlined here? https://github.com/conveyal/gtfs2json/blob/007dc123acf655a2f06ad3b529facb9d77809434/LICENSE

Reply:
Done, but it's from `https://github.com/node-geojson/gtfs2geojson`, not `https://github.com/conveyal/gtfs2json/blob/007dc123acf655a2f06ad3b529facb9d77809434/LICENSE`